.aws/task-definition-prod.json

```diff
@@ -41,6 +41,26 @@
         {
           "name": "MONGODB_PASSWORD",
           "valueFrom": "/tis/revalidation/prod/db/password"
+        },
+        {
+          "name": "TIS_DBHOST",
+          "valueFrom": "/tis/ndw/prod/source/db/host"
+        },
+        {
+          "name": "TIS_DBPORT",
+          "valueFrom": "/tis/ndw/prod/source/db/port"
+        },
+        {
+          "name": "TIS_DBNAME",
+          "valueFrom": "/tis/ndw/prod/source/db/name"
+        },
+        {
+          "name": "TIS_DBUSER",
+          "valueFrom": "/tis/ndw/prod/source/db/user"
+        },
+        {
+          "name": "TIS_DBPASSWORD",
+          "valueFrom": "/tis/ndw/prod/source/db/password"
```

Comment:
We shouldn't really use "ndw" parameters.  It might be more appropriate to move them to parameters with different names if they can be shared.  I'm most keen to identify whether then user:password combination used here is one that can be shared across ETLs

Reply:
Good point. I have made the changes.